common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1232,6 +1235,15 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("10s")
 
+  val NETWORK_MEMORY_ALLOCATOR_ALLOW_CACHE: ConfigEntry[Boolean] =
+    buildConf("celeborn.network.memory.allocator.allowCache")
+      .categories("network")
+      .internal
+      .version("0.4.0")
```

Comment:
maybe 0.3.1, if @waitinfuture not oppose to

Reply:
No opposition:)

Reply:
The graph looks great!